packages/veritone-react-common/src/components/TopBar/styles.scss

```diff
@@ -7,6 +7,7 @@
   justify-content: space-between;
   left: 0;
   right: 0;
+  margin-bottom: 20px;
```

Comment:
this should be removed -- it doesnt make sense for a self contained component like this to impose a margin on the content below it. also it will mess up all other existing users of topbar.

Reply:
👍